components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/configuration/models/SdkLanguageConfigurations.java

```diff
@@ -0,0 +1,31 @@
+package org.wso2.carbon.apimgt.core.configuration.models;
```

Comment:
Is this auto generated class? If not please add licence header

Reply:
This is not auto generated. Please add the license header.